dev/release/verify-release-candidate.sh

```diff
@@ -99,14 +100,7 @@ fetch_archive() {
   shasum -a 512 -c ${dist_name}.tar.gz.sha512
 }
 
-test_binary() {
-  local download_dir=binaries
-  mkdir -p ${download_dir}
-
-  python3 $SOURCE_DIR/download_rc_binaries.py $VERSION $RC_NUMBER --dest=${download_dir}
-
-  pushd ${download_dir}
-
+verify_cwd_artifact_signatures() {
```

Comment:
The refactor make this function side effect free. Does it need to be cwd, can it just take the path as input?

Reply:
done